### PR TITLE
Add handling for unhandled exceptions

### PR DESCRIPTION
**Pull request by ryoe**
Address Issue #1 by adding TransformController changes to handle XmlTransformationException (and XmlNodeException). Also add minimal error handling in Index.cshtml transform().

### CODE REVIEW

**Comment by friism** [comment]
Awesome!, thanks for the pull request - I've been meaning to look at this for a long time.

It'd be great if you could use tabs for indentation, as is used in the rest of the code. You can make Visual Studio show white space by hitting ctrl+r, ctrl+w.

File: AppHarbor.TransformTester/Controllers/TransformController.cs [hunk 1]
```diff
@@ -12,42 +12,47 @@ public class TransformController : Controller
...
+                        document.WriteTo(xmlTextWriter);
+                    }
+                    return Content(stringBuilder.ToString(), "text/xml");
+                }
+                else
+                {
+                    return ErrorXml("Transformation failed for unkown reason");
+                }
+            }
+            catch (XmlTransformationException ex)
```
Comment (friism) [inline AppHarbor.TransformTester/Controllers/TransformController.cs]: consider naming the exception variable `xmlTransformationException`

File: AppHarbor.TransformTester/Controllers/TransformController.cs [hunk 2]
```diff
@@ -12,42 +12,47 @@ public class TransformController : Controller
...
+            {
+                return ErrorXml(ex.Message);
+            }
+            catch (XmlException exception)
```
Comment (friism) [inline AppHarbor.TransformTester/Controllers/TransformController.cs]: consider naming the exception variable `xmlException`

**Comment by ryoe** [comment]
Thanks for the tip on Ctrl-R, Ctrl-W! I had to tell VS to Keep Tabs too. I'll do all of the things you suggested. What is the preferred way? Should I do a new pull request with the tabs issue and the variable renames?

**Comment by friism** [comment]
Whatever works for you, your can amend the commit (easy if you're using Git Gui) and do `git push -f`. Github will update the PR automatically as you change the branch.

File: AppHarbor.TransformTester/Controllers/TransformController.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public ActionResult Create(string webConfigXml, string transformXml)
 					var stringBuilder = new StringBuilder();
 					var xmlWriterSettings = new XmlWriterSettings();
 					xmlWriterSettings.Indent = true;
-					xmlWriterSettings.IndentChars = "    ";
+					xmlWriterSettings.IndentChars = "	";
 					using (var xmlTextWriter =
```
Comment (ryoe) [inline AppHarbor.TransformTester/Controllers/TransformController.cs:27]: Should this have been 4 spaces instead of a tab?

Comment (friism) [inline AppHarbor.TransformTester/Controllers/TransformController.cs:27]: At any rate, it's an unrelated change, so consider omitting the change.
